### PR TITLE
Fix odd back of links caught by link checker

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: en/config_fw/advanced_tuning_guide_fixedwing.md [hunk 1]
```diff
@@ -79,9 +79,9 @@ Furthermore, these two values define the height rate limits commanded by the use
 
 ### L1 Controller Tuning (Position)
```
Comment (hamishwillee) [inline en/config_fw/advanced_tuning_guide_fixedwing.md:80]: @sfuhrer Can you propose the docs you'd like to see here? 

Based on your other fix it is the stuff here http://docs.px4.io/main/en/advanced_config/parameter_reference.html#fw-npfg-control

What is NPFG?

File: en/frames_vtol/vtol_tiltrotor_eflite_convergence_pixfalcon.md [hunk 2]
```diff
@@ -24,19 +24,19 @@ Flight controller options ():
 
 ## Hardware Setup
 
-The vehicle needs 7 PWM signals, which must be connected to the flight controller outputs as specified in [Airframe Reference > VTOL Tiltrotor > E-flite Convergence](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_e-flite_convergence).
-This mapping is reproduced below. 
+The vehicle needs 7 PWM signals, which must be connected to the flight controller outputs as specified in [Airframe Reference > VTOL Tiltrotor > E-flite Convergence](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_e-flite_convergence).
```
Comment (hamishwillee) [inline en/frames_vtol/vtol_tiltrotor_eflite_convergence_pixfalcon.md:27]: @sfuhrer The eflight doesn't have a config any more. Would you just start from the second tailsitter option under https://docs.px4.io/main/en/airframes/airframe_reference.html#vtol-tiltrotor ?

This airframe doc doesn't say anything about the motors or support for elevons. I "imagine" you can map all the actuators as shown below, but can't try it. Can you confirm whether this might be possible using normal Actuator configuration steps. Any "gotchas"?

Basically I'm after enough information to know whether this can still be done, and to enable a beginner to do it, if they have the Actuator Config screen.

File: en/config_fw/advanced_tuning_guide_fixedwing.md (hunk 1 above)
Comment (hamishwillee) [reply en/config_fw/advanced_tuning_guide_fixedwing.md:80]: PS Note, I have tagged bits that need your attention. Ignore anything not tagged to you.

Comment (sfuhrer) [reply en/config_fw/advanced_tuning_guide_fixedwing.md:80]: I'd like to leave that to @tstastny . NPFG is the new FW guidance that replaced the L1 controller. How to tune it stays largely the same though I would say.

File: en/frames_vtol/vtol_tiltrotor_eflite_convergence_pixfalcon.md (hunk 2 above)
Comment (hamishwillee) [reply en/frames_vtol/vtol_tiltrotor_eflite_convergence_pixfalcon.md:27]: Thanks. I'll update appropriately!

```suggestion
The vehicle needs 7 PWM signals for the motors and control surfaces:
- Motor (left/right/back)
- Tilt servos (right/left)
- Elevons (left/right)

These can be configured in the [Actuators Configuration](../config/actuators.md) by following the instructions for VTOL tiltrotor geometry and output configuration.
Note that you will need to start from the [Generic Tiltrotor VTOL](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_generic_tiltrotor_vtol) frame.
```

File: en/frames_vtol/vtol_tiltrotor_eflite_convergence_pixfalcon.md [hunk 3]
```diff
@@ -1,74 +1,80 @@
...
+MAIN 1 | Motor right
+MAIN 2 | Motor left
+MAIN 3 | Motor back
+MAIN 4 | empty
+MAIN 5 | Tilt servo right
+MAIN 6 | Tilt servo left
+MAIN 7 | Elevon right 
+MAIN 8 | Elevon left 
+
+Note that left and right in the reference are defined from the perspective of a human pilot inside a real plane (or looking from above, as shown below):
```
Comment (hamishwillee) [inline en/frames_vtol/vtol_tiltrotor_eflite_convergence_pixfalcon.md:47]:
```suggestion

Note that left and right in the configuration screen and frame reference are defined from the perspective of a human pilot inside a real plane (or looking from above, as shown below):

```

File: en/frames_vtol/vtol_tiltrotor_eflite_convergence_pixfalcon.md [hunk 4]
```diff
@@ -1,74 +1,69 @@
...
+- Tilt servos (right/left)
+- Elevons (left/right)
+
+These can be configured in the [Actuators Configuration](../config/actuators.md) by following the instructions for VTOL tiltrotor geometry and output configuration.
+Note that you will need to start from the [Generic Tiltrotor VTOL](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_generic_tiltrotor_vtol) frame.
+
+Note that left and right in the configuration screen and frame reference are defined from the perspective of a human pilot inside a real plane (or looking from above, as shown below):
+
+
+<img src="../../assets/airframes/types/VTOLTiltRotor_eflite_convergence.svg" width="300px" />
```
Comment (hamishwillee) [inline en/frames_vtol/vtol_tiltrotor_eflite_convergence_pixfalcon.md:38]:
```suggestion
## Hardware Setup

The vehicle needs 7 PWM signals for the motors and control surfaces:
- Motor (left/right/back)
- Tilt servos (right/left)
- Elevons (left/right)

These can be wired to flight controller outputs more-or-less as you desire.

The outputs and configured in the [Actuators Configuration](../config/actuators.md) by following the instructions for VTOL tiltrotor geometry and output configuration.
Note that you will need to start from the [Generic Tiltrotor VTOL](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_generic_tiltrotor_vtol) frame.

Note that left and right in the configuration screen and frame reference are defined from the perspective of a human pilot inside a real plane (or looking from above, as shown below):

<img src="../../assets/airframes/types/VTOLTiltRotor_eflite_convergence.svg" width="300px" />

```

File: en/config_fw/advanced_tuning_guide_fixedwing.md (hunk 1 above)
Comment (hamishwillee) [reply en/config_fw/advanced_tuning_guide_fixedwing.md:80]: @tstastny  I'm merging this PR because it fixes lots of problems. Can you create a separate PR to address this specific section?
